vnext/JSI/Shared/PreparedJavaScriptStore.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+namespace facebook::jsi {
+class PreparedJavaScript;
+}
+
+namespace Microsoft::JSI {
+
+/**
+ * TODO (yicyao)
+ */
+class PreparedJavaScriptStore {
+ public:
```

Comment:
>public [](start = 1, length = 6)

I would recommend creating a small test which takes advantage of this Api. It could become much easier to spot which options should be exposed to the user.